case/src/main/kotlin/com/ritense/case/domain/CaseTab.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.case.domain
+
+import javax.persistence.EmbeddedId
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.Table
+
+@Entity
+@Table(name = "case_tab")
+class CaseTab(
+
+    @EmbeddedId
+    val id: CaseTabId,
+
+    val name: String,
+
+    val tabOrder: Int,
+
+    @Enumerated(EnumType.STRING)
+    val type: CaseTabType,
+
+    val content: String,
```

Comment:
Can we have a property name that better describes what's in there? From the examples, it looks more like a component-key or something

Reply:
The issue is that this can be multipole things depending on the type. It's either the form key, system tab name or angular component name.

Reply:
`contentKey` then? `content` gives me vibes of a JSON-object, which is not what seems to be intended.